tests/storage/misc_test.py

```diff
@@ -415,9 +415,9 @@ def testValidInput(self):
         """
         Test that when all arguments are correct the method works smoothly.
         """
-        writeData = (b"DON'T THINK OF IT AS DYING, said Death. "
-                     b"JUST THINK OF IT AS LEAVING EARLY TO AVOID THE RUSH.")
-        # (C) Terry Pratchet - Good Omens
+        writeData = (b"Lorem ipsum dolor sit amet, consectetur adipiscing "
+                     b"elit. Curabitur risus turpis, gravida vel ligula in, "
+                     b"rhoncus maximus sem. Curabitur ac.")
```

Comment:
Some data, why do we need random data?

Reply:
I was just trying to figure out what did you mean by "some data" where "Lorem ipsum" is not good enough. I understand that it does not have to be a text-like input, but I still need to write something and assign it to `writeData`. So, I could smash my keyboard, generate data with a (deterministic) data generator in the test itself, or generate/look for random data and then paste it in the code.

Or maybe you are happy enough by just doing:
```python
        writeData = b"Some data"
```
I am not sure, and that's what I was trying to ask.

Reply:
Yes, this looks good.

Check if the test is expecting certain size - it is not clear what this test is trying to do, and the copyrighted text did not help to understand what the tests is checking.

If the test is expecting data with certain size, the best way to create it is:

    b"x" * expected_size

If the test just need some data with any size:


    b"some data"

is makes the tests very simple to understand and debug in case the test fails.

If the test is about blocks, it can be useful to create blocks of text:

    b"A" * 512 + b"B" * 512

Finally, it is useful to see how vdsm is using this code, and test in the same
way. For example when we write metadata, we create a block of 512 bytes like this:

    b"data".